src/websocket_server/server.ts

```
@@ -0,0 +1,43 @@
+    ws.on('message', (message) => {
+        let clientMsg: ClientMessage;
+        try {
+            clientMsg = JSON.parse(message.toString());
+        } catch (err) {
+            console.error(`[${connectionId}] Invalid JSON received.`);
+            return;
+        }
+
+        switch (clientMsg.type) {
+            case 'reg':
+                let regData: RegClientData;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)